testdata/scripts/health/multi-chain.txtar

```diff
@@ -75,14 +75,14 @@ ok EVM.1.Txm.BlockHistoryEstimator
 ok EVM.1.Txm.Broadcaster
 ok EVM.1.Txm.Confirmer
 ok EVM.1.Txm.WrappedEvmEstimator
+ok HeadReporterService
```

Comment:
We need to align on whether this is a global service, or an EVM service.

Reply:
We also must find a solution that will continue to work with the LOOP Plugin runtime, namely one which does not depend on subscribing to head trackers from multiple chains, which will be running in separate LOOPPs.

Reply:
With your pr it remains as global service.
Will reconsider this approach with loop/chain_reader later

Reply:
Moved to global service + global config